hack/test-update-storage-objects.sh

```diff
@@ -103,7 +103,7 @@ echo "${ETCD_VERSION}" > "${ETCD_DIR}/version.txt"
 
 # source_file,resource,namespace,name,old_version,new_version
 tests=(
-test/e2e/testing-manifests/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
+test/e2e/testing-manifests/rbd-storage-class.yaml storageclasses slow v1beta1 v1
```

Comment:
change reason:
```
In hack/test-update-storage-objects.sh line 105:
tests=(
      ^-- SC2054: Use spaces, not commas, to separate array elements.
```

Reply:
I suspect we want this to be one element.